disneyVillains.sql

```diff
@@ -0,0 +1,56 @@
+-- MySQL dump 10.13  Distrib 8.0.24, for macos11 (x86_64)
+--
+-- Host: localhost    Database: villains
+-- ------------------------------------------------------
+-- Server version	8.0.24
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!50503 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
```

Comment:
This is not what your sql file should look like.  When I replaced it with the file from your OTHER repo with some tweaks I got your project to work.  This is what it looks like
drop database villains;
create database villains;
USE villains;
DROP USER 'VillainsUser'@'localhost';
create user 'VillainsUser'@'localhost' IDENTIFIED WITH mysql_native_password BY 'Dannyboy12345!';
GRANT ALL ON villains.* TO 'VillainsUser'@'localhost';

CREATE TABLE `disneyVillains` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(255) DEFAULT NULL,
  `movie` varchar(255) DEFAULT NULL,
  `slug` varchar(255) DEFAULT NULL,
  `createdAt` datetime DEFAULT CURRENT_TIMESTAMP,
  `updatedAt` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  `deletedAt` datetime DEFAULT NULL,
  PRIMARY KEY (`id`)
);


INSERT INTO `disneyVillains` VALUES (1,'Captain','Peter Pan','captain-hook','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(2,'Cruella de vil','One Hundred and one Damaltians','cruella-de-vil','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(3,'Gaston','Beauty and the Beast','gaston','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(4,'Hades','Hercules','hades','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(5,'Horned King','The Black Cauldron','horned-king','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(6,'Jafar','Alladin','jafar','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(7,'Lady Tremaine','Cinderella','lady-tremaine','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(8,'Madam Medusa','The rescuers','madame-medusa','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(9,'Madam Mim','The Sword in the Stone','madame-mim','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(10,'Maleficent','Sleeping Beauty','maleficent','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(11,'Prince John','Robin Hood','prince-john','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(12,'Sir Hiss','Robin Hood','sior-hiss','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(13,'Queen Grimhilde','Snow White and the Seven Dwarfs','queen-grimhilde','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(14,'Queen of Hearts','Alice in Wonderland','queen-of-hearts','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(15,'Scar','The Lion King','scar','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(16,'Shan Yu','Mulan','shan-yu','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(17,'Shere Khan','The Jungle Book','shere-khan','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL),(18,'Ursula','The Little Mermaid','Ursula','2021-05-09 14:09:10','2021-05-09 14:09:10',NULL);